### PR TITLE
Adding alternative text in the pictures to the contribution file

### PR DESCRIPTION
**Pull request by CaduRoriz**
-   [ ] Fixes or refactors
-   [ ] A new challenge
-   [ x ] Additional documentation
-   [ ] Something else

#### Description
Alternative text has been added to the images in the contribution document.

### Relations

Relates #1049 

### Checklist:

-   [ ] All the contributions made are solely the work of me and my co-authors
-   [x ] I tested the changes in this PR (if applicable)
-   [ ] I added unit tests to ensure my change works (when change in Java or on front-end code)
-   [ ] I added UI tests to ensure my UI changes work (when change in the overall UI, not needed if just adding a challenge)
-   [ ] The PR passes pre-commit hooks and automated tests

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -213,44 +213,44 @@ Further use the **_OWASP WrongSecrets --> Lifecycle --> install_** step to load
 
 Open the **_WrongSecretsApplication_** by following the path **_main>java>org.owasp.wrongsecrets>WrongSecretApplication_**.
 
-![](images/open-application-6.1.png)
+![Click on the 'WrongSecretsApplication' file located at main > java > org.owasp.wrongsecrets > WrongSecretApplication](images/open-application-6.1.png)
```
Comment (commjoen) [inline CONTRIBUTING.md:216]: Can you please note down all the settings changed in each picture?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -213,44 +213,44 @@ Further use the **_OWASP WrongSecrets --> Lifecycle --> install_** step to load
...
-![](images/run-application-6.2.png)
+![Click the 'Run' button or press Shift + F10](images/run-application-6.2.png)
 
 ### There you have it, **_WrongSecrets_** running successfully.
 
 Here is a _preview_ on how does it look after successfully running the Application.
 **Note:** Running the Application doesn't open any kind of **_GUI_**, it only initializes the **_local webserver_** that you can open via a **_browser._**
 
-![](images/final-output-8.png)
+![Command line terminal running successfully](images/final-output-8.png)
```
Comment (commjoen) [inline CONTRIBUTING.md:249]: Can you describe the logs and the chosen profiles?

**Review by commjoen — CHANGES_REQUESTED** [review]
Hi @CaduRoriz ! Looks great! Can you extend the logging picture its alt text please? We need at least the profile and the last sentence indicating a successful boot.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -213,44 +213,44 @@ Further use the **_OWASP WrongSecrets --> Lifecycle --> install_** step to load
...
 
 **_Select_** all the fields that are Selected in the below picture.
 
-![](images/select-options-7.3.png)
+![In the Run Options select: 'Enviroment variables' | 'Add VM options' | 'Shorten command line' | 'Specify classes and packages' | 'Open run/debug tool window when started'](images/select-options-7.3.png)
 
 **_Fill out_** all the fields as shown below.
 
-![](images/fill-fields-7.4.png)
+![Fill out the fields: | Build and run: org.owasp.wrongsecrets.WrongSecretsApplication : -Dserver.port=8080 - Dspring.profiles.active=local,without-vault. | Working directory: /Users/razr/workspace/owasp/wrongsecrets | Environment variables: K8S_ENV=docker| Packages and classes to include in covarege data: org.owasp.wrongsecrets.*| ](images/fill-fields-7.4.pn)
```
Comment (commjoen) [inline CONTRIBUTING.md:238]: Can you explain that the working directory depends on the enviroment setup of the user?

Comment (commjoen) [reply CONTRIBUTING.md:238]: It might be a good idea to describe which setting goes where :-)

Comment (CaduRoriz) [reply CONTRIBUTING.md:238]: i'll do it :)

**Pull request merged**